src/index.ts

```diff
@@ -50,6 +50,21 @@ export interface WorkspacePermissionsSpec {
   public: boolean;
 }
 
+export interface Workspace {
+  id: number;
+  name: string;
+  created: string;
+  modified: string;
+  arango_db_name: string;
+}
+
+export interface WorkspacesSpec {
+  count: number;
+  next: number;
+  previous: number;
+  results: Workspace[];
+}
```

Comment:
I actually think having a generic interface for any response like this (which is very common in our api) is beneficial. 
```suggestion
export interface Paginated<T> {
  count: number,
  next: string,
  previous: string,
  results: T[],
}
```
I wouldn't actually commit this suggestion, instead I'd recommend just making this change locally, with the addition of replacing uses of `WorkspaceSpec` with `Paginated<Workspace>`.

Reply:
That makes sense- will do!

Reply:
Looks like this is addressed by 27be0fa, thanks!